tests/test_views.py

```diff
@@ -88,6 +94,20 @@ def setUp(self):
         self.link.save()
         self.project_url = self.project.web_url_for('view_project')
 
+    def test_edit_title_empty(self):
```

Comment:
Would you mind moving these tests to `test_views/TestProjectViews`? It looks like this is testing the project title, and not view only links.